app/assets/scss/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$diff-insertion-color: #acf2bd;
```

Comment:
A small part of me wants these in here - https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/scss/_digital-marketplace-colours.scss

It feels like 'diff' colours are something we have used in more than one place so could standardise them there.

Although I'm not too sure if this suggestion is actually too much work for the benefit. I'll let you make the call @DilwoarH